queries/subsquid/bsx/offerListUser.graphql

```diff
@@ -0,0 +1,16 @@
+query offerListUser($id: String!) {
+  offerEvents(where: { caller_eq: $id }) {
+    offer {
```

Comment:
Seem we do not need `offerEvents`.
There is a data named  `createdAt` in `offer` query.